### ISSUE TITLE
Read on array of int doesn't work

### ISSUE THREAD

**Issue by Wilders** [Ang3/php-odoo-api-client#6]
Hi !

I'm trying to read on multiple ids to get a result for each ids :

This code throw an RequestException:
```php
$client->read('stock.move', $myarray, ['fields' => ['product_id']]);
```

I've got the same problem on any other models & any other arrays

I've tried to use array_values but it doesn't work.

It works with only one id.

#0 C:\laragon\www\api-prod\vendor\ang3\php-odoo-api-client\src\Client.php(336): Ang3\Component\Odoo\Endpoint->call('execute_kw', Array)
#1 C:\laragon\www\api-prod\vendor\ang3\php-odoo-api-client\src\Client.php(143): Ang3\Component\Odoo\Client->call('stock.move', 'read', Array, Array)

Do you have any idea ?

So I'm actually iterating over the array of int and getting one by one the results.. But it takes so much long time this way.

Thanks

**Comment by Ang3** [comment]
Hi,

Thank you for this feedback. Indeed, Odoo expects a value of type ```array<array<int>>``` was just casting your array as array... I fixed the client in version ```v6.1.3```. You just have to update the package via composer, I tested with integers and arrays. ;)

Note that you could use the method ```findBy``` to do the job.

Thanks. ;)

**Comment by Wilders** [comment]
Thanks for the fast fix, it works great 👍🏼
